modules/kubernetes/src/test/java/org/apache/ignite/discovery/KubernetesDiscoveryAbstractTest.java

```diff
@@ -101,42 +78,59 @@ public void testClientConnectsToCluster() throws Exception {
 
         cfg.setDiscoverySpi(discoverySpi);
 
-        cfg.setIgniteInstanceName(getTestIgniteInstanceName());
+        cfg.setGridLogger(log);
+
+        cfg.setIgniteInstanceName(instanceName);
+        cfg.setClientMode(clientMode);
 
         return cfg;
     }
 
-    /** */
-    private void mockServerResponse(String... addrs) {
+    /**
+     * Mock HTTP server responce
+     *
+     * @param addrs Address list to return.
+     */
+    protected final void mockServerResponse(String... addrs) {
+        mockServerResponse(1, addrs);
+    }
+
+    /**
+     * Mock HTTP server responce
+     *
+     * @param times How many times mock should return value.
+     * @param addrs Address list to return.
+     */
+    protected final void mockServerResponse(int times, String... addrs) {
         String ipAddrs = Arrays.stream(addrs)
-            .map(addr -> String.format("{\"ip\":\"%s\"}", addr))
-            .collect(Collectors.joining(","));
+                .map(addr -> String.format("{\"ip\":\"%s\"}", addr))
```

Comment:
What's the reason behind reformatting? 4 spaces is a proper padding according to our code style guidelines.